packages/vite-node/package.json

```diff
@@ -74,7 +74,7 @@
     "*.mjs"
   ],
   "engines": {
-    "node": ">=v14.18.0"
+    "node": "^18.0.0 || >=20.0.0"
```

Comment:
1. Why can't we use `>=18.0.0`? 
2. We should probably use 18.12.0 as it was the first LTS release (https://nodejs.org/ru/blog/release/v18.12.0)

Reply:
@sheremet-va  I copy+paste from Vite (see [here](https://github.com/vitejs/vite/blob/40fd2d9bf4073420e6c334f48dc3b63558b688ce/packages/vite/package.json#L49-L51)), I don't mind changing it though, I'm not sure why they decided to go with `^18.0` instead of `>=18`, I would also prefer the latter. Just tell me what you want to use and I could update it

Reply:
Vite uses `^18.0.0 || >=20.0.0` to explicitly say Vite doesn't support Node 19 (EOLed in this june).

Reply:
thanks @sapphi-red I keep forgetting about these odd versions, @sheremet-va I have reapplied the same as Vite. Ready for final review. Thanks